baselines/common/vec_env/test_vec_env.py

```diff
@@ -67,6 +67,24 @@ def make_fn(seed):
     assert_venvs_equal(env1, env2, num_steps=num_steps)
 
 
+@pytest.mark.parametrize('dtype', ('uint8', 'float32'))
+def test_sync_sampling(dtype):
+    num_envs = 4
+    num_envs_in_series = 2
+    num_steps = 100
+    shape = (3, 8)
+
+    def make_fn(seed):
+        """
```

Comment:
this is a little shameful how in baselines we have docstrings for single-line functions like this, and no docstrings for monster functions with 15 parameters with obscure names :) Given that the test right above this code is the same way, I leave it to your discretion as to whether to keep the docstring or remove it. Let me put it like this - if the docstring is not there, I won't complain :)